packages/picker/src/Picker.ts

```diff
@@ -417,6 +425,13 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
         `;
     }
 
+    public renderDescriptionSlot(): TemplateResult {
+        return html`
+            <div id=${ifDefined(this.description)}>
```

Comment:
This value is internal, please make it static. Doing so should allow us to remove the `description` property from this class as well.

Reply:
Once this value is static, please add documentation for this slot similar to [this](https://github.com/adobe/spectrum-web-components/blob/main/packages/picker/src/Picker.ts#L689).

Reply:
Done!